Compilation errors under Nix

When compiling under Nix environment we get the following error, doesn't matter if its building the gem in development or when compiling the gem when installing. 

```
Activating xapian-core 1.4.22 (from ./xapian-full/ports/aarch64-apple-darwin24.2.0/xapian-core/1.4.22)...
Extracting xapian-bindings-1.4.22.tar.xz into tmp/aarch64-apple-darwin24.2.0/ports/xapian-bindings/1.4.22... OK
Running 'configure' for xapian-bindings 1.4.22... OK
Running 'compile' for xapian-bindings 1.4.22... ERROR, review './xapian-full/tmp/aarch64-apple-darwin24.2.0/ports/xapian-bindings/1.4.22/compile.log' to see what happened. Last lines are:
========================================================================
      _wrap_LatLongCoords_size(int, unsigned long*, unsigned long) in xapian_wrap.o
      _wrap_Database_size(int, unsigned long*, unsigned long) in xapian_wrap.o
      _wrap_Database_get_total_length(int, unsigned long*, unsigned long) in xapian_wrap.o
      _wrap_Database_get_revision(int, unsigned long*, unsigned long) in xapian_wrap.o
      _wrap_Database_check(int, unsigned long*, unsigned long) in xapian_wrap.o
  "_rb_undef_alloc_func", referenced from:
      _Init__xapian in xapian_wrap.o
  "_rb_undef_method", referenced from:
      _Init__xapian in xapian_wrap.o
  "_rb_unexpected_type", referenced from:
      _Init__xapian in xapian_wrap.o
ld: symbol(s) not found for architecture arm64
clang++: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [Makefile:741: _xapian.la] Error 1
make[3]: Leaving directory './xapian-full/tmp/aarch64-apple-darwin24.2.0/ports/xapian-bindings/1.4.22/xapian-bindings-1.4.22/ruby'
make[2]: *** [Makefile:670: all] Error 2
make[2]: Leaving directory './xapian-full/tmp/aarch64-apple-darwin24.2.0/ports/xapian-bindings/1.4.22/xapian-bindings-1.4.22/ruby'
make[1]: *** [Makefile:514: all-recursive] Error 1
make[1]: Leaving directory './xapian-full/tmp/aarch64-apple-darwin24.2.0/ports/xapian-bindings/1.4.22/xapian-bindings-1.4.22'
make: *** [Makefile:436: all] Error 2
========================================================================
rake aborted!
```

--- a/Rakefile
+++ b/Rakefile
@@ -28,6 +28,7 @@ task :core do
 end
 
 task bindings: [:core] do
+  ENV['LIBS'] = RbConfig::CONFIG["LIBRUBYARG"]
   ENV['RUBY_LIB'] = ENV['RUBY_LIB_ARCH'] = File.join(__dir__, 'lib')
 
   MiniPortile.new("xapian-bindings", ver).tap do |recipe|

